Articles/Blog/2023-03-08-productivity-practices-and-tools.md

```diff
@@ -0,0 +1,155 @@
+Productivity journaling captures tasks, observations, ideas, and lessons to be periodically reviewed, distilled and migrated.
+
+* **Tasks**: identify something that needs to be done (e.g. a bug to fix).
+* **Observations**: describe something that happened worth remembering almost like in a lab journal.
+* **Ideas**: capture unspecific but maybe important thoughts to be processed later (e.g. a better approach to a sorting problem).
+* **Lessons**: record key takeaways worth sharing with others (even your future self).
+
+Effective productivity journaling also demands that one periodically slow down, pause, reflect and review their record of activity.
+Such pauses enable one to make more deliberate decisions (e.g. acting instead of re-acting) about future work as well as to recognize patterns of activity worth changing.
+While there are many approaches to journaling, two useful sets of strategies come from [Bullet Journaling by Ryder Carroll](https://bulletjournal.com/) and [Building A Second Brain by Tiago Forte](https://www.buildingasecondbrain.com/)
+
+#### Key Journaling Strategies
+
+What to journal and how to journal are deeply personal.
+Here are a few of the high level strategies to consider:
+
+First, treat a journal as an append only log.
+Journaling is most useful when one can make entries quickly and easily.
+The more effort involved, the less likely it will be done.
+Now, one may ask, *"if the journal is append only, and everything is jumbled together, how can you ever find anything?"*
+The answer is in the second key strategy.
+
+Second, periodically reflect and distill what was journaled.
+Sort through the material to organize what is truly important and copy into more permanent homes.
+Bullet journaling recommends using pen and paper journals, because it requires effort to physically copy things from one place to another.
+
+Common reflection cadences are daily, weekly and monthly.
+Where appropriate, it might be worthwhile to align reflection cadence with a project's sprint cadence.
+[Building a Second Brain](https://www.buildingasecondbrain.com/) recommends *progressive summarization* which involves underlining important entries upon daily review, highlighting important underlined entries upon weekly review and migrating (copying) important highlighted entries upon monthly review.
+
+At longer cadences, more substantial reflection and review is appropriate.
+For example, at monthly intervals, its reasonable to check in with yourself and ask...what should I stop, start, and continue for next month?
+Are my practices productive and effective or do I need to make some adjustments?
+At quarterly intervals, ask yourself...what is my vision/theme/aspiration(s) for this season?
+How well did I live up to my aspirations last quarter?
+
+#### Tools to consider
+
+There is a video series from [Tiago Forte](https://www.youtube.com/playlist?list=PLVNXAaej57W7fbYoc_XJ0bA0T3k_Ye-Q2) that describes a variety of journaling tools.
```

Comment:
```suggestion
There is a video series from [Tiago Forte](https://www.youtube.com/playlist?list=PLVNXAaej57W7fbYoc_XJ0bA0T3k_Ye-Q2) that describes which journaling tools work best for different kinds of person.
```

i like breaking these tools out per section.  However the video describes more than just a variety of tools, but how to find a tool that will work best for you depending on your "note taking archetype", a concept that I thought was important, but just didn't quite make the cut in my original post.

Reply:
Fixed I think in a37507247a57d4368f35665f2cfc59e23a162612